configuration/errors.go

```diff
@@ -0,0 +1,13 @@
+package configuration
+
+import "fmt"
+
+// ErrCannotParseEnvVar is returned if an environment variable cannot be parsed
+type ErrCannotParseEnvVar struct {
+	EnvVar string
+	Err    error
+}
+
+func (err ErrCannotParseEnvVar) Error() string {
+	return fmt.Sprintf("Error parsing environment variable %s: %v", err.EnvVar, err.Err)
```

Comment:
```suggestion
	return fmt.Sprintf("parsing environment variable %s: %v", err.EnvVar, err.Err)
```

Non-blocking: Starting with 'error', 'failed', etc at the start of an error tends to result in bad final error strings like `failed xx failed xx failed xx`